instrumentation/kamon-play/src/main/scala/kamon/instrumentation/play/PlayServerInstrumentation.scala

```diff
@@ -239,3 +246,13 @@ object GenerateOperationNameOnFilterHandler {
   })
 
 }
+
+object GenerateGRPCOperationName {
+  @Advice.OnMethodEnter
+  def enter(@Advice.Argument(0) request: akka.http.scaladsl.model.HttpRequest): Unit = {
+    val span = Kamon.currentSpan()
+    span.name(request.uri.toRelative.toString)
```

Comment:
Little nitpicking: we usually use the fluid API when doing several "things" on the same Span!